packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -125,18 +136,46 @@ export function Table<T>(props: TableProps<T>): ReactElement {
                 },
                 Cell: ({ cell, value }) =>
                     props.cellRenderer(
-                        (children: ReactNode) => (
-                            <div
-                                {...cell.getCellProps()}
-                                {...(!props.columnsResizable ? { style: { flex: "1 1 0px" } } : {})}
-                                className="td"
-                            >
-                                {children}
-                            </div>
-                        ),
+                        (children: ReactNode) => {
+                            const content = (
+                                <div {...cell.getCellProps()} className="td">
+                                    {children}
+                                </div>
+                            );
+
+                            /**
+                             * Checks if column is not the only one visible to set size
+                             */
+                            if (cell.row.index === 0 && cell.column.width === "autoFit" && cell.row.cells.length > 1) {
```

Comment:
Can it be simpler achieved by always having a resizer with `handleWidth={cell.column.width === "autoFit" && cell.row.cells.length > 1}`

Reply:
I want explicitly to add the references only for the first row to trigger resize in the header when it changes

Reply:
But if there are no rows, aren't we in trouble?

Reply:
Solved 😛